### ISSUE TITLE
GLideN64 LLE bug

### ISSUE THREAD

**Issue by Nejsplitter** [gonetz/GLideN64#2699]
When using a combination of GLideN64 RDP and ParaLLEl RSP the lights/lens flares dont render in Perfect Dark

Tested on Mupen64Plus-AE and Mupen64Plus-Libretro-nx

**Comment by Xii-Nyth** [comment]
Good catch, just tested it in rmg. Will have to check this in other games since theres really no reason to do such a thing for pd.
![image](https://user-images.githubusercontent.com/101597311/186238727-6c6b6872-98fd-4d72-9255-c0a0f6ff391a.png)
